GitHub-Test-4/nlp_exercise_1.py

```diff
@@ -0,0 +1,33 @@
+import spacy
+
+nlp = spacy.load('en_core_web_sm')
```

Comment:
I'd recommend keeping the download line in your code, just comment it out after the first run. That way, you're less likely to forget to add it in a new file. It could sit like this:
`
#nlp = spacy.cli.download("en_core_web_md") 
`
`
nlp = spacy.load('en_core_web_md')
`